hack/mock-unittest-data/gomod.py

```
@@ -0,0 +1,154 @@
+            where="workspaces/go_list_deps_threedot.json",
+            what="go list -deps -json=ImportPath,Module,Standard,Deps ./..."
+        )
+        run_redirect(where="workspaces/go_work.json", what="go work edit -json")
+
+        update_workspaces(
+            run=run, run_redirect=run_redirect, pandemo_dir=pandemo_dir,
+            mocked_data_dir_abspath=mocked_data_dir_abspath, mocked_data_dir=mocked_data_dir
+        )
+        run(_c("git restore ."))
+        run(_c("git switch main"))
+        run_redirect(where=f"non-vendored/go_list_modules.json", what="go list -m -json")
```

Ruff (0.14.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)